plugins/source/digitalocean/resources/plugin/plugin.go

```diff
@@ -0,0 +1,33 @@
+package plugin
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/source/digitalocean/client"
+	"github.com/cloudquery/plugin-sdk/plugins"
+)
+
+var (
+	Version = "development"
+)
+
+// todo change it
```

Comment:
There's a todo here, not sure if it should be removed? 

Also as an aside, in some other plugins we've adopted the convention of storing the config in a yml file and embedding with `go:embed`; the main advantage being that you get syntax highlighting when editing the file.

I'd also love to see a comment above each line, so that it's clear what is optional and what is not.